libsignal-service/Cargo.toml

```diff
@@ -43,3 +43,4 @@ tokio = { version = "1.0", features = [ "macros" ] }
 
 [features]
 prefer-e164 = []
+require-send = []
```

Comment:
I'm not sure whether `require-send` is the most appropriate name for the feature.

Additionally, it may be useful to document this in the README somehow.